docs/en/enterprise-edition/content-collections/runtime-security/agentless-scanning/configure-accounts/configure-azure.adoc

```diff
@@ -21,33 +21,56 @@ Complete the steps below to configure agentless scanning based on the needs of y
 +
 image::runtime-security/agentless-azure-configuration.png
```

Comment:
was the updated screenshots from the docs added to all CSPs? this seems broken in the current live version

Reply:
Yes, I updated the screenshots for all the CSPs in this PR.

Reply:
The question is do we want the screenshot here or after the next step of expanding the advanced settings. WDYT?

Reply:
Moved.